common/scripted_triggers/missions_expanded_buildings_scripted_triggers.txt

```diff
@@ -79,8 +79,9 @@ can_forcelimit_line = {
 
 province_has_building_of_group = {
 	#	"group" that can be checked:
-	#		building groups (trade, government, navy, production, army, taxation, defense, all) -- all does NOT include manufactories
-	#
+	#		building groups (trade, government, navy, production, army, taxation, defense, all) 
+	#		-- government does not use universities
```

Comment:
Manufactory types added to the `all` list, University removed from the government list. See inline comment below.